tasks/html.js

```diff
@@ -58,11 +58,11 @@ function setUpTaskHtml(tyRun) {
       });
     },
     onRun: function onRunHtmlTask() {
-      var runSequence = require('run-sequence');
-      runSequence(taskDefinition.name);
+      var gulp        = options.gulp || require('gulp');
+      gulp.start.apply(gulp, [taskDefinition.name]);
     }
   };
-  tyRun.taskYargs.register(taskDefinition);
+  options.taskYargsRun.taskYargs.register(taskDefinition);
```

Comment:
As you know my preference is to eventually move the `taskDefinition` to be the return value (if not module exports). I believe the plugin engine should encapsulate the task yargs functionality.

Would this break generality in the plugin registry package?